docs/index.js

```diff
@@ -60,5 +62,8 @@ swagger.paths['/notifications/{id}'] = updateNotification;
 swagger.paths['/articles/trends'] = trendingArticles;
 
 swagger.paths['/articles/{slug}/share/mail'] = shareArticle;
+swagger.paths['/statistics/published'] = articlesReadCount;
+swagger.paths['statistics/read'] = myReadCount;
```

Comment:
Hey Victor, there should be a  starting / here

Reply:
Thank you very much. I will see to that.